library/alloc/src/ffi/c_str.rs

```diff
@@ -643,6 +643,9 @@ impl CString {
     /// Runtime checks are present to ensure there is only one nul byte in the
     /// [`Vec`], its last element.
     ///
+    /// If the nul byte may not be at the end, [`CString::from_vec_until_nul`]
+    /// can be used instead.
+    ///
```

Comment:
We shouldn't recommend unstable functions from the docs of stable functions, since most people can't use those yet.

Reply:
Fixed.